install.py

```diff
@@ -221,16 +221,10 @@ def build_project(project, extra_args):
         shutil.rmtree(os.path.join(proj_dir, "build"))
 
     print("Installing {!r} using {!r}".format(project, sys.executable))
-    status = subprocess.call(
+    subprocess.check_call(
```

Comment:
I don't like this change, the old code nicely prints which subproject fails to install and the new one doesn't.